docs/code/code-getting-started.asciidoc

```diff
@@ -1,7 +1,7 @@
 [[code-getting-started]]
 == Getting Started with Code
```

Comment:
We've been trying to get away from using "Getting Started" when talking about features.  Is there other language you can use?

Reply:
Do you have an example of other language being used for a similar zero to hero style page? I have a couple of ideas but it would be good to make sure I'm being consistent with the rest of the doc.

Reply:
Some suggestions:

- Follow what [Canvas](http://localhost:8000/guide/canvas.html) does
- Put the GS stuff on the first page

Since this pattern appears throughout the Kibana docs, let's discuss with the other writers and make the change at once.